README.md

```diff
@@ -1,6 +1,17 @@
 # Airship
 > Airship is a components library built by CARTO
 
+[![CircleCI](https://circleci.com/gh/CartoDB/airship/tree/master.svg?style=svg)](https://circleci.com/gh/CartoDB/airship/tree/master)
+
+There are two npm packages, for easier reuse:
+
+
+| module | status | version | description |
+|---|---|---|---|
+|  [airship-style](https://www.npmjs.com/package/@carto/airship-style) | ![status](https://img.shields.io/badge/status-in%20progress-orange.svg) | [![npm version](https://badge.fury.io/js/%40carto%2Fairship-style.svg)](https://badge.fury.io/js/%40carto%2Fairship-style)| A complete CSS framework to build location intelligence apps. |
+|  [airship-components](https://www.npmjs.com/package/@carto/airship-components) | ![status](https://img.shields.io/badge/status-in%20progress-orange.svg) | [![npm version](https://badge.fury.io/js/%40carto%2Fairship-components.svg)](https://badge.fury.io/js/%40carto%2Fairship-components) | Reusable and easy to use web components. |
```

Comment:
I liked the reference to LI apps in the styles package. Add it here also, please.

`... web components for Location Intelligence apps.`.